app/tests/integration/nimbus/pages/home.py

```diff
@@ -7,7 +7,7 @@ class HomePage(Page):
     """Nimbus Home page."""
 
     _create_new_btn_locator = (By.CSS_SELECTOR, "#create-new-button")
-    _page_wait_locator = (By.CSS_SELECTOR, ".table")
+    _page_wait_locator = (By.CSS_SELECTOR, ".directory-table")
```

Comment:
I might not be understanding the exact role of `data-testid`.
Above we have the following snippet:
```html
<table className="table" data-testid="DirectoryTable">
```

I don't know I'd expect this CSS selector to work if we're only setting a `.table` class, but then trying to use `.directory-table` here. Not sure if the selector should be something weird like `table.table[data-testid="DirectoryTable"]` or if just `[data-testid="DirectoryTable"]` works here (or heck, even just `.table` since we're now following The Highlander rules of "there can be only one").

But now that we've split the 3 sub-tables on live into separate tabs, the tests might need to be tweaked further if we were trying to do a lookup of experiments in "Review" or "Preview" states. /cc @jrbenny35